frontend/src/components/layout/Header/Header.tsx

```diff
@@ -76,38 +77,28 @@ const Header = () => {
       </Modal.Footer>
     </Modal>
   );
-  //example of how to check role: const isAdmin = keycloak?.realmAccess?.roles.includes(USER_ROLES[Permission.ADMIN]) == true;
   return (
-    <Navbar fixed="top" expand="xl" className="App-header">
-      <Container className="bar" fluid={true}>
-        <Row>
-          <Col>
-            <Row className="brand-box">
-              <Col md={2} lg={2}></Col>
-              <Col xs={8} sm={6} md={4} lg={1} className="brand">
-                <Navbar.Brand onClick={() => history.push('/')}>
-                  <img
-                    className="bc-gov-icon"
-                    src={logoUrl}
-                    width="156"
-                    height="43"
-                    alt="Go to the Government of British Columbia website"
-                  />
-                </Navbar.Brand>
-              </Col>
-              <Col xs={3} sm={4} md={2} lg={7} className="title">
-                <h1 className="longAppName">Property Inventory Management System</h1>
-                <h1 className="shortAppName">PIMS</h1>
-              </Col>
-            </Row>
-          </Col>
-          <Col xs={3} sm={3} md={3} lg={1} className="other">
-            {errors && errors.length ? (
-              <FaBomb size={30} className="errors" onClick={handleShow} />
-            ) : null}
-          </Col>
-        </Row>
-      </Container>
+    <Navbar expand className="App-header">
+      <Navbar.Brand className="brand-box" onClick={() => history.push('/')}>
+        <img
+          className="bc-gov-icon"
+          src={logoUrl}
+          width="156"
+          height="43"
+          alt="Go to the Government of British Columbia website"
+        />
+      </Navbar.Brand>
+      <Nav className="title mr-auto">
+        <Nav.Item>
+          <h1 className="longAppName">Property Inventory Management System</h1>
+          <h1 className="shortAppName">PIMS</h1>
+        </Nav.Item>
+      </Nav>
+      <Nav className="other">
+        {errors && errors.length ? (
+          <FaBomb size={30} className="errors" onClick={handleShow} />
+        ) : null}
+      </Nav>
```

Comment:
When a `<Navbar/>` is expanded it already shows nav items in a row, so I remove some extra markup to simplify the component